doc/release/0.17.0-notes.rst

```diff
@@ -29,6 +29,11 @@ Deprecated features
 Backwards incompatible changes
 ==============================
 
+The functions ``trim1`` and ``trimboth`` now partially (for numpy >= 1.8.0) or
```

Comment:
Can you make this ``stats.trim1`` and ``stats.trimboth``? Otherwise they may be hard to find.